components/src/dynamo/vllm/omni/omni_handler.py

```
@@ -111,6 +187,198 @@ def __init__(
+                        await unregister_model(
+                            endpoint=self.generate_endpoint,
+                            lora_name=lora_name,
+                        )
+                    except Exception as unreg_err:
+                        logger.exception(
+                            "Failed to unregister LoRA '%s' from discovery",
+                            lora_name,
+                        )
+                        yield {
+                            "status": "error",
+                            "message": f"Failed to unregister LoRA '{lora_name}' from discovery registry: {str(unreg_err)}",
```

Ruff (0.15.20)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)